test/framefusion.test.ts

```diff
@@ -354,7 +354,37 @@ describe('FrameFusion', () => {
         // Act & assert
         // ensure we render the last few frames properly - we have to flush the decoder to get the last few frames
         for (let i = 20; i < 30; i++) {
-            const time = i / FPS + FRAME_SYNC_DELTA;
+            const time = i / FPS;
```

Comment:
I would have liked to see a few more tests with different FPS which are multiples of 5 up to 120, but maybe we should not worry about this until we have a case that actually proves that this is an issue 🤔

Reply:
Maybe it's not even an issue, so just ignore my comment 😆